src/components/Map/Map.js

```diff
@@ -572,7 +572,12 @@ export default class Map extends React.Component<Props, State> {
 
     this.props.mappingEvents &&
       this.props.mappingEvents
-        .filter(event => event.status === 'ongoing' || event._id === this.props.featureId)
+        .filter(
+          event =>
+            event.status === 'ongoing' ||
+            event.status === 'planned' ||
+            event._id === this.props.featureId
```

Comment:
Unrelated: featureId is no longer only a featureId, if I read this correctly?

Reply:
Good catch: This might be a leftover inconsistency since in the beginning a `MappingEvent` was in indeed a `Feature`. It isn't anymore. There is now a `MappingEventFeature` type. I will probably clean this up in the current trello ticket were I clean up event-related typings